lib/ontologies_linked_data/metrics/metrics.rb

```diff
@@ -33,12 +33,10 @@ def self.metrics_for_submission(submission, logger)
         metrics.properties = prop_count
         logger.info("properties finished")
         logger.flush
-
         # re-generate metrics file
-        submission.generate_metrics_file2(cls_metrics[:classes], indiv_count, prop_count, cls_metrics[:maxDepth])
+        submission.generate_metrics_file(cls_metrics[:classes], indiv_count, prop_count)
```

Comment:
discuss with bioportal about this generate_metrics_file2 (method) for now stay with the old one